### PR TITLE
Rapid Breach Response | Added FireEye playbook

### PR DESCRIPTION
**Pull request by altmannyarden**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
will update the release notes after this pr will merge:
https://github.com/demisto/content/pull/10567

## Description
Added a new playbook for the pack, the new playbook will search for the breached FireEye red team tools IOCs.

## Screenshots
image

## Minimum version of Demisto
- [ ] 5.0.0
- [x] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/MajorBreachesInvestigationandResponse/Playbooks/playbook-FireEye_Red_Team_Tools_Investigation_and_Response.yml [hunk 1]
```diff
@@ -0,0 +1,1242 @@
...
+          "y": 3060
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "24":
+    id: "24"
```
Comment (David-BMS) [inline Packs/MajorBreachesInvestigationandResponse/Playbooks/playbook-FireEye_Red_Team_Tools_Investigation_and_Response.yml:1018]: Consider switching to findIncidentsWithIndicator
Provides a much neater output and context and without the bug with the strange output (worth opening a separate issue for it).

File: Packs/MajorBreachesInvestigationandResponse/Playbooks/playbook-FireEye_Red_Team_Tools_Investigation_and_Response.yml [hunk 2]
```diff
@@ -0,0 +1,1242 @@
...
+    nexttasks:
+      '#none#':
+      - "19"
+    scriptarguments:
+      EndpointId: {}
+      Hostname:
+        complex:
+          root: Endpoint
+          accessor: HostName
+      IP:
```
Comment (David-BMS) [inline Packs/MajorBreachesInvestigationandResponse/Playbooks/playbook-FireEye_Red_Team_Tools_Investigation_and_Response.yml:846]: Need to make sure your not providing external IP addresses to block
Consider using something like IsIPInRanges with a playbook input that contains the relevant ranges
The Splunk Indicator Hunting can be used as a reference.

File: Packs/MajorBreachesInvestigationandResponse/Playbooks/playbook-FireEye_Red_Team_Tools_Investigation_and_Response.yml [hunk 3]
```diff
@@ -0,0 +1,1242 @@
...
+          "x": 130,
+          "y": 620
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "6":
```
[the remark this answers is not in the document]
Comment (altmannyarden) [reply Packs/MajorBreachesInvestigationandResponse/Playbooks/playbook-FireEye_Red_Team_Tools_Investigation_and_Response.yml:379]: The subplaybook doesnt now enrich the CVEs automatically

**Comment by ShirleyDenkberg** [comment]
@David-BMS @michalgold Doc review completed.

File: Packs/MajorBreachesInvestigationandResponse/Playbooks/playbook-FireEye_Red_Team_Tools_Investigation_and_Response.yml [hunk 4]
```diff
@@ -0,0 +1,1242 @@
...
+    skipunavailable: false
+    quietmode: 0
+  "1":
+    id: "1"
+    taskid: e32d2c90-ec6b-4e07-8943-944df5ae8d2f
+    type: condition
+    task:
+      id: e32d2c90-ec6b-4e07-8943-944df5ae8d2f
+      version: -1
+      name: Is there CVE's?
```
[the remark this answers is not in the document]
Comment (David-BMS) [reply Packs/MajorBreachesInvestigationandResponse/Playbooks/playbook-FireEye_Red_Team_Tools_Investigation_and_Response.yml]: @ShirleyDenkberg Shouldn't this be "Are there"?

**Review by David-BMS — APPROVED** [review]

**Pull request merged**